htdocs/fichinter/class/api_interventions.class.php

```diff
@@ -207,6 +207,7 @@ public function index($sortfield = "t.rowid", $sortorder = 'ASC', $limit = 100,
 					if ($contact_type) {
 						$fichinter_static->contacts_ids = $fichinter_static->liste_contact(-1, $contact_type, 1);
 					}
+					$fichinter_static->fetchObjectLinked();
```

Comment:
The fetchObjectLinked is a heavy load operation, this is why it is not done by default when getting a long list.
To follow the same behaviour of other message (like api_order), can you add a parameter $loadlinkedobjects
and then add a condition to call the fetchObjectLinked()

if ($loadlinkedobjects) {
						// retrieve linked objects
						$commande_static->fetchObjectLinked();
					}